default.nix

```diff
@@ -257,4 +257,90 @@ rec {
   shell = pkgs.mkShellNoCC {
     packages = [ ];
   };
+
+  demo-system =
+    module:
+    let
+      nixosSystem =
+        args:
+        import (sources.nixpkgs + "/nixos/lib/eval-config.nix") (
+          {
+            inherit lib;
+            system = null;
+          }
+          // args
+        );
+    in
+    nixosSystem {
+      system = "x86_64-linux";
+      modules = [
+        module
+        (sources.nixpkgs + "/nixos/modules/profiles/qemu-guest.nix")
+        (sources.nixpkgs + "/nixos/modules/virtualisation/qemu-vm.nix")
+        (
+          { config, ... }:
+          {
+            users.users.nixos = {
+              isNormalUser = true;
+              extraGroups = [ "wheel" ];
+              initialPassword = "nixos";
+            };
+
+            users.users.root = {
+              initialPassword = "root";
+            };
+
+            security.sudo.wheelNeedsPassword = false;
+
+            services.getty.autologinUser = "nixos";
+            services.getty.helpLine = ''
+
+              Welcome to Ngipkgs!
+            '';
+
+            virtualisation = {
```

Comment:
I suggest to move `virtualisation` configuration under `services.openssh` and I am good with this PR. Great job !